packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -245,12 +244,7 @@ function TokenRow({ style, onClick, token }: TokenRowProps): JSX.Element {
                 })
               ) : (
                 <div className="mr-2">
-                  <Loader
-                    type="Oval"
-                    color="rgb(40, 160, 240)"
-                    height={14}
-                    width={14}
-                  />
+                  <Loader color="rgb(40, 160, 240)" size="small" />
```

Comment:
NIT: I know it wasn't added by this PR, but unless we need opacity on the background, I would only use hex colors